src/mgr/ActivePyModule.cc

```diff
@@ -100,6 +100,61 @@ void ActivePyModule::notify_clog(const LogEntry &log_entry)
   }
 }
 
+bool ActivePyModule::method_exists(const std::string &method) const
+{
+  Gil gil(py_module->pMyThreadState, true);
+
+  auto boundMethod = PyObject_GetAttrString(pClassInstance, method.c_str());
```

Comment:
might want to avoid the camelCase naming convention.

Reply:
good point - updated